libhif/hy-util.c

```diff
@@ -147,52 +150,28 @@ hy_detect_arch(char **arch)
 #undef MAX_ARCH_LENGTH
 
 int
-hy_split_nevra(const char *nevra, char **name, long int *epoch,
+hy_split_nevra(const char *nevra, char **name, int *epoch,
```

Comment:
@cgwalters @hughsie are you ok with this API change? In other places `epoch` is represented as `int`. But we don't insist on this change if it could cause problems.